packages/components/src/Input/Input.spec.js

```diff
@@ -0,0 +1,61 @@
+import { getByTestId, render } from '@testing-library/react';
+import { ThemeProvider, createTheme } from '@platzily-ui/styling';
+import Input from './Input';
+
+describe('@Components/Input', () => {
+  it('Given the Input Component, when the props provide an object within the styles attribute then the component will take those styles ', () => {
+    // arrange
+    const theme = createTheme();
+    const { getByRole } = render(
+      <ThemeProvider theme={theme}>
+        <Input role="banner" style={{ color: theme.palette.secondary.main }} />
+      </ThemeProvider>,
+    );
+
+    // act
+    const InputTestedText = getByRole('banner');
+    // assert
+    expect(InputTestedText).toBeDefined();
+    expect(InputTestedText).toHaveStyle(`color:#97c343`);
+  });
+
+  it('Given the Input Component, when the props provide type or required attributes then the component will take those props', () => {
+    // arrange
+    const { getByRole } = render(
+      <ThemeProvider theme={createTheme()}>
+        <Input role="form" type={'date'} required />
+      </ThemeProvider>,
+    );
+
+    // act
+    const InputTestedText = getByRole('form');
+
+    // assert
+    expect(InputTestedText).toBeDefined();
+    expect(InputTestedText).toHaveProperty('type');
```

Comment:
:thinking: what are we validating here? What about if we assert that the type property is date?